libs/template/presets/fulfillment/experience/pages/login-page.ts

```diff
@@ -13,9 +12,8 @@ export const fulfillmentLoginPage: ExperienceComponent = {
     rules: [
       {
         layout: {
-          ...(featureVersion >= '1.4'
-            ? { type: 'list' }
-            : { type: 'flex', vertical: true }),
+          type: 'flex',
+          vertical: true,
```

Comment:
I'd prefer to keep this as is, and fix the list layout styles. 

We should add the following to the list styles:

```css

      align-items: var(--align);
      align-content: var(--align);
      justify-content: var(--justify);
      justify-items: var(--justify);
```